README.md

```diff
@@ -118,6 +118,16 @@ Sample setting:
 "checker-framework.languageserver_repo": "checker-framework-languageserver"
 ```
 
+
+## LSP-related Command Line Options
+This section provides a list of the Checker Framework command line options that are related to the features of this LSP 
+plugin. You can turn on/off an option by adding/removing the corresponding flag in 
+[`commandLineOptions`](#commandlineoptions).
+
+- `-AlspTypeInfo`: Once enabled, the plugin will display a pop-up showing the related type information when hovering on 
```

Comment:
Should https://github.com/eisopux/checker-framework-vscode/blob/4e85b1ad2fffaff400981d320f41edc365bc7a90/package.json#L66 pass this option by default? Or at least contain it commented-out to make it easier to discover?

Reply:
Should be fine to pass the option by default. I have updated package.json.